pycbc/events/stat.py

```diff
@@ -1204,6 +1204,24 @@ def assign_median_sigma(self, ifo):
         self.fits_by_tid[ifo]['median_sigma'] = \
             coeff_file['median_sigma'][:][tid_sort]
 
+    def lognoiserate(self, trigs, alphabelow=6):
+        """Calculate the log noise rate density over single-ifo newsnr
+
+        Read in single trigger information, make the newsnr statistic
+        and rescale by the fitted coefficients alpha and rate
+        """
+        alphai, ratei, thresh = self.find_fits(trigs)
+        newsnr = self.get_newsnr(trigs)
+        bt = newsnr > thresh
```

Comment:
is this > or < ?

Reply:
should be < (this has swapped a couple of times during development)